### PR TITLE
fix article control buttons

### LINKED ISSUE #728: Fix button styling
When logged in as admin, there are too many buttons to fit on the articles page and the buttons are squashed.

photo_2021-11-08 20 40 59

### PR DESCRIPTION
**Pull request by faissaloux**
resolves #728

https://user-images.githubusercontent.com/60013703/140835062-41a8434d-4ccb-4e37-ad2d-335653468e0e.mp4

### CODE REVIEW

**Comment by driesvints** [comment]
I like this. Can we make all buttons like this? Also pin and delete.

I also think we need different icons for publish and decline. Maybe these two?

Approve: https://blade-ui-kit.com/blade-icons/heroicon-s-check
Decline: https://blade-ui-kit.com/blade-icons/heroicon-s-x

File: resources/views/components/articles/actions.blade.php [hunk 1]
```diff
@@ -1,58 +1,97 @@
-<div class="flex flex-col gap-x-4 gap-y-3 lg:flex-row lg:justify-between">
+<div
+    class="flex flex-col gap-x-4 gap-y-3 lg:flex-row lg:justify-between"
+    x-data="{ hovered:false }"
```
Comment (joedixon) [inline resources/views/components/articles/actions.blade.php:3]:
```suggestion
    x-data="{ hovered: false }"
```

File: resources/views/components/articles/actions.blade.php [hunk 2]
```diff
@@ -1,58 +1,97 @@
...
+                @mouseover="hovered = 'pin'"
+                @mouseleave="hovered = false"
+                :selected="$article->isPinned()"
+                class="w-full"
+            >
                 <span class="flex items-center gap-x-2">
-                    <x-icon-pin class="w-5 h-5" title="{{ $article->isPinned() ? 'Unpin' : 'Pin' }}"/> 
-                    {{ $article->isPinned() ? 'Unpin article' : 'Pin article' }}
+                    <x-icon-pin class="w-5 h-5" title="{{ $article->isPinned() ? 'Unpin' : 'Pin' }}"/>
+                        <span x-show="hovered === 'pin'">{{ $article->isPinned() ? 'Unpin article' : 'Pin article' }}</span>
```
Comment (joedixon) [inline resources/views/components/articles/actions.blade.php:78]:
```suggestion
                    <span x-show="hovered === 'pin'">{{ $article->isPinned() ? 'Unpin article' : 'Pin article' }}</span>
```

**Review by joedixon — CHANGES_REQUESTED** [review]
I really like this too, great job. Just a couple of minor formatting tweaks from me.

**Comment by joedixon** [comment]
@driesvints I'm happy with this when you are

**Comment by driesvints** [comment]
Thanks @faissaloux!

**Comment by driesvints** [comment]
I just noticed this PR causes a weird issue with buttons expanding on the bottom padding when you hover over them. Can we get that fixed?

**Comment by faissaloux** [comment]
Yes @driesvints I just noticed the same issue I'll fix it

**Comment by faissaloux** [comment]
@driesvints #732 fixes this height issue

**Comment by driesvints** [comment]
@faissaloux I just noticed that I can't seem to click any of the buttons anymore in Safari after this.

**Comment by joedixon** [comment]
@driesvints fixed here https://github.com/laravelio/laravel.io/pull/747

**Pull request merged**